cmd/okctl/apply_application.go

```diff
@@ -42,10 +42,6 @@ func buildApplyApplicationCommand(o *okctl.Okctl) *cobra.Command {
 		Args:  cobra.ExactArgs(requiredApplyApplicationArguments),
 		PreRunE: func(cmd *cobra.Command, args []string) error {
 			err := o.Initialise()
-			if err != nil {
```

Comment:
Why is this removed?

Reply:
Oh, no, it shouldnt be, probably a mistake from the merging. Strange that wasmt caught by linting

Reply:
I fixded